pkg/endpoint/bpf.go

```diff
@@ -712,6 +712,20 @@ func (e *Endpoint) regenerateBPF(owner Owner, epdir, reason string) (uint64, boo
 		}
 	}
 
+	// Walk the L4Policy to add new redirects and update the desired policy map
```

Comment:
The changes to this file look suspicious. I would revert just those.

Reply:
OK I'll drop the patch then.
https://github.com/cilium/cilium/pull/5405/commits/ea5ec53175d5783c44f90a50cc830b2bfdcaf1c4

Reply:
Yes, that's the one. Thanks!